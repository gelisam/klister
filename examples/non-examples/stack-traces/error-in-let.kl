#lang "prelude.kl"

(define fail (lambda (thing)
               (let (go (+ thing (error 'Im-an-error)))
                 go)))

(example (fail 23))
