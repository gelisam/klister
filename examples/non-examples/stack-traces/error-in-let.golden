In Phase: p0:
    error-in-let.kl:4.42-4.53: Im-an-error
    stack trace:
    ---- with function #<+>
    ---- in let #[error-in-let.kl:4.22-4.24]<go>
    ---- Halt
